docs/en/stack/ml/df-analytics/dfa-classification.asciidoc

```diff
@@ -4,70 +4,100 @@
 
 experimental[]
 
-{classification-cap} is a {ml} process for predicting the class or category of a 
-given data point in a dataset. Typical examples of {classification} problems are 
-predicting loan risk, classifying music, or detecting cancer in a DNA sequence. 
-In the first case, for example, our dataset consists of data on loan applicants 
-that covers investment history, employment status, debit status, and so on. 
-Based on historical data, the {classification} analysis predicts whether it is 
-safe or risky to lend money to a given loan applicant. In the second case, the 
-data we have represents songs and the analysis – based on the features of the 
-data points – classifies the songs as hip-hop, country, classical, or any 
-other genres available in the set of categories we have. Therefore, 
-{classification} is for predicting discrete, categorical values, unlike 
-{reganalysis} which predicts continuous, numerical values.
-
-From the perspective of the possible output, there are two types of 
-{classification}: binary and multi-class {classification}. In binary 
-{classification} the variable you want to predict has only two potential values. 
-The loan example above is a binary {classification} problem where the two 
-potential outputs are `safe` or `risky`. The music classification problem is an 
-example of multi-class {classification} where there are many different potential 
-outputs; one for every possible music genre. In the {version} version of the 
-{stack}, you can perform only binary {classanalysis}.
-
-
-[discrete]
-[[dfa-classification-features]]
-==== {feature-vars-cap}
-
-When you perform {classification}, you must identify a subset of fields that you 
-want to use to create a model for predicting another field value. We refer to 
-these fields as _{feature-vars}_ and _{depvar}_, respectively. 
-{feature-vars-cap} are the values that the {depvar} value depends on. There are 
-three different types of {feature-vars} that you can use with our 
-{classification} algorithm: numerical, categorical, and boolean. Arrays are not 
-supported in the {feature-var} fields.
+{classification-cap} is a {ml} process that enables you to predict the class or
+category of a data point in your data set. Typical examples of {classification}
+problems are predicting loan risk, classifying music, or detecting cancer in a
+DNA sequence. In the first case, for example, the data set might contain the
```

Comment:
Hello everyone! 👋 
Very minor issue, but I would consider rewording this sentence `detecting cancer in a
DNA sequence` to something like `detecting the potential for cancer from a DNA sequence` , because technically cancer is an illness in a tissue/organ and not in the DNA sequence.